pom.xml

```diff
@@ -177,6 +184,14 @@
                 <scope>test</scope>
             </dependency>
 
+            <!-- remoting3 -->
+            <dependency>
+                <groupId>org.jboss.remoting3</groupId>
+                <artifactId>jboss-remoting</artifactId>
+                <version>${version.org.jboss.remoting3}</version>
+                <scope>test</scope>
+            </dependency>
+
             <!-- External tests -->
```

Comment:
I think only this block (and the definition of ${version.org.jboss.remoting3}) is really necessary.
I just added this and ran dependency:tree and that seems to be enough for all submodules.

Could you remove all other changes and try with only this?